src/radical/pilot/configs/resource_ornl.json

```diff
@@ -1,93 +1,55 @@
-        "description"                 : "The Cray XK7 supercomputer located at the Oak Ridge Leadership Computing Facility (OLCF), (https://www.olcf.ornl.gov/titan/)",
-        "notes"                       : "Requires the use of an RSA SecurID on every connection.",
-        "schemas"                     : ["local", "ssh", "go"],
+    "andes": {
+        "description"                 : "704 compute nodes",
+        "notes"                       : "Requires RSA SecurID and uses local virtual env",
+        "schemas"                     : ["local"],
         "local"                       : {
-            "job_manager_endpoint"    : "slurm://rhea.ccs.ornl.gov/",
-            "filesystem_endpoint"     : "file://rhea.ccs.ornl.gov/"
-        },
-        "ssh"                         : {
-            "job_manager_endpoint"    : "slurm+ssh://rhea.ccs.ornl.gov",
-            "filesystem_endpoint"     : "sftp://rhea.ccs.ornl.gov/"
-        },
-        "go"                          : {
-            "job_manager_endpoint"    : "slurm+ssh://rhea.ccs.ornl.gov",
-            "filesystem_endpoint"     : "go://olcf#dtn/"
+            "job_manager_endpoint"    : "slurm://andes.olcf.ornl.gov/",
+            "filesystem_endpoint"     : "file://andes.olcf.ornl.gov/"
         },
+        "forward_tunnel_endpoint"     : "andes.olcf.ornl.gov",
         "default_queue"               : "batch",
-        "resource_manager"            : "TORQUE",
-        "agent_type"                  : "multicore",
-        "agent_config"                : "rhea",
+        "resource_manager"            : "SLURM",
         "agent_scheduler"             : "CONTINUOUS",
         "agent_spawner"               : "POPEN",
-        "agent_launch_method"         : "APRUN",
-        "task_launch_method"          : "APRUN",
-        "mpi_launch_method"           : "APRUN",
-        "pre_bootstrap_0"             : [
-            "module purge",
-            "module load PE-gnu",
-            "module load python/3.7.0",
-            "module load py-setuptools",
-            "module load py-virtualenv",
-            # Workaround for ZMQ runtime failure
-            "export LD_PRELOAD=/lib64/librt.so.1"
-        ],
-        "default_remote_workdir"      : "$MEMBERWORK/`groups | cut -d' ' -f2`",
-        "valid_roots"                 : ["/lustre/atlas/scratch"],
-        "rp_version"                  : "local",
-        "virtenv_mode"                : "use",
-        "virtenv"                     : "/lustre/atlas2/%(pd.project)s/world-shared/ve.rp.titan.2018.05.23",
+        "agent_launch_method"         : "SRUN",
+        "task_launch_method"          : "SRUN",
+        "mpi_launch_method"           : "SRUN",
+        "pre_bootstrap_0"             : ["source /sw/andes/python/3.7/anaconda-base/etc/profile.d/conda.sh"],
+        "valid_roots"                 : ["$MEMBERWORK/"],
+        "default_remote_workdir"      : "$MEMBERWORK/%(pd.project)s",
+        "python_dist"                 : "local",
```

Comment:
Sorry, didn't see this on the first read.  `local` will only reliably work if
  - the client virtualenv is on a shared filesystem (usable from the compute nodes)
  - the client virtualenv can be activated using the module loads above (so, if the user happens to use the exact same modules)
I would find this too risky for a production config (which we let end users use), and think it's safer to let the bootstrapper set up the env.  The overhead is not large enough to matter in production.  What do you think?
Same for summit (just more so ;-) )

Reply:
all that points work and we don't force user to use a specific type of virtenv or a specific name

> think it's safer to let the bootstrapper set up the env

this case requires to have a pre-set virtenv (since there is no connection to outside from compute nodes), but I would propose to extend using such approach for other similar cases (to be discussed during devel-meeting and will be a separate ticket/PR)